cirq/google/vis/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
this should import directly the public classes and methods you want to expose.  We try to encourage people to use 

```
import cirq
cirq.Blah 
```
 in contrast to
```import cirq.google.vis as cgv
cgv.Blah.```
This works well for libraries like cirq (following numpy and tensorflow here).

Reply:
Done.